### PR TITLE
Add a PULL_REQUEST_TEMPLATE to make contribution clearer

### PR DESCRIPTION
**Pull request by jnothman**
@adrinjalali would this have helped?

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+If you are adding a docset, please ensure that:
+
+* [ ] the title of this pull request names the project and version being added.
```
Comment (Kapeli) [inline PULL_REQUEST_TEMPLATE.md]: As the only person that reviews pull requests, I don't see a need for this.

**Comment by Kapeli** [comment]
This feels like adding bureaucracy just for the sake of it, to solve an issue that occurred only 1 time in the last few years. 

I'm sorry @adrinjalali wasted his time updating a version string and submitting a pull request, but if I force everyone to read this every time they submit a pull request, a lot more time would be wasted, collectively.

**Comment by jnothman** [comment]
Do what you wish. Adrin was not upset by the time spent making a pr, he was
upset by the unfamiliar experience of having a pull request closed if it
did not meet requirements, especially when those requirements were not
clearly stated in advance in any place he had thought to look.

The purpose of this pr is to make sure requirements are clearly stated.

 If you are unconcerned by the title of pull requests I'm happy to remove
that requirement.

**Comment by Kapeli** [comment]
I'm sorry, but I don't think this pull request template adds any value to this repo.

I wasn't aware of any GitHub etiquette that disallows closing pull requests. If this issue occurs again, I'll consider making a pull request template that clearly states that pull requests that can't be merged will be closed.

**Pull request closed without merging**